### PR TITLE
Added news item on the 14th PinT workshop (2024), and updated the 2024 workshop information page.

### PR DESCRIPTION
**Pull request by ignace-computing**

### CODE REVIEW

File: _events_upcoming/2024-13th-pint-workshop.md [hunk 1]
```diff
@@ -1,25 +1,35 @@
...
+  - name: Stefan Vandewalle
+    homepage: https://www.hsu-hh.de/hpc/philipp-neumann/
+- name: Stefan Vandewalle
+    homepage: https://www.hsu-hh.de/hpc/philipp-neumann/
+invited:
+  - name: Peter Düben (ECMWF)
+    homepage: https://www.ecmwf.int/en/about/who-we-are/staff-profiles/peter-dueben
+  - name: Stefanie Günther (LLNL)
+    homepage: https://scholar.google.com.sg/citations?user=C1tJycgAAAAJ&hl=en
+    homepage: https://www.researchgate.net/profile/Stefanie-Guenther-4
```
Comment (pancetta) [inline _events_upcoming/2024-13th-pint-workshop.md:23]: Pretty sure this does not work. Just one homepage, please.

File: _events_upcoming/2024-13th-pint-workshop.md [hunk 2]
```diff
@@ -1,25 +1,35 @@
...
----
-
-PinT 2024: the 13th Workshop on Parallel-in-time Integration will be hosted in Bruges, Belgium by Dr. Giovanni Samaey.
-Conference website:  https://time-x.eu/pint-2024/
-
-
-
-
 
+PinT 2024: the 14th Workshop on Parallel-in-time Integration will be hosted in Bruges, Belgium, by Professor Giovanni Samaey, professor Stefan Vandewalle and Ward Melis.
```
Comment (pancetta) [inline _events_upcoming/2024-13th-pint-workshop.md:33]: Typo in "professor" (capitalize)

File: _events_upcoming/2024-13th-pint-workshop.md [hunk 3]
```diff
@@ -1,25 +1,35 @@
...
+    homepage: https://www.researchgate.net/profile/Stefanie-Guenther-4
+  - name: Johan Meyers (KUL)
+    homepage: https://www.kuleuven.be/wieiswie/en/person/00011045
+  - name: Sebastian Schöps (TUD)
+    homepage: https://www.cem.tu-darmstadt.de/cem/group/schops.en.jsp
 permalink: /events/13th-pint-workshop/
 subtitle: "Brief overview"
 page_type: event_page
 no_lead: true
----
```
Comment (pancetta) [inline _events_upcoming/2024-13th-pint-workshop.md:17]: Don't remove the dashes.

**Comment by ignace-computing** [comment]
Dear @pancetta 
Thank you for spotting these issues, and apologies for the lack of care from my side.
Hopefully the commit fc880d2 can solve these issues.
Good day,
Ignace

**Pull request merged**